roles/prometheus/defaults/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+prometheus_version: 1.8.0
+prometheus_gomaxprocs: "{{ ansible_processor_vcpus|default(ansible_processor_count) }}"
+prometheus_service_reload: true
+
+node_exporter_version: 0.15.0
+node_exporter_release_name: "node_exporter-{{ node_exporter_version }}.linux-amd64"
+node_exporter_pid_path: /var/run/node-exporter.pid
+
+alertmanager_version: 0.9.1
+alertmanager_release_name: "alertmanager-{{ alertmanager_version }}.linux-amd64"
+alertmanager_pid_path: "/var/run/alertmanager.pid"
+
+pushgateway_version: 0.4.0
+pushgateway_release_name: "pushgateway-{{ pushgateway_version }}.linux-amd64"
+pushgateway_pid_path: "/var/run/pushgateway.pid"
+
+grafana_config_dir: /etc/grafana
+
+prometheus_release_name: "prometheus-{{ prometheus_version }}.linux-amd64"
+
+prometheus_root_dir: /opt/prometheus
+prometheus_dist_dir: "{{ prometheus_root_dir }}/dist"
+prometheus_bin_dir: "{{ prometheus_root_dir }}/current"
+prometheus_rules_dir: "{{ prometheus_config_dir }}/rules"
+prometheus_file_sd_config_dir: "{{ prometheus_config_dir }}/tgroups"
+
+prometheus_config_dir: /etc/prometheus
+prometheus_console_libs_dir: /etc/prometheus/console_libraries
+prometheus_console_dir: /etc/prometheus/consoles
+prometheus_log_dir: /var/log/prometheus
+prometheus_pid_path: /var/run/prometheus.pid
+prometheus_db_dir: /var/lib/prometheus
+
+prometheus_web_listen_address: ":9090"
+prometheus_web_external_url: 'http://localhost:9090/'
+prometheus_alertmanager_url: 'http://localhost:9093'
+
+prometheus_config_flags:
```

Comment:
It's easier to hardcode all this in the unit file, we don't need these configurable